src/_rules/border.js

```diff
@@ -24,6 +24,15 @@ export const borders = [
   [/^border()-(.+)$/, handlerBorderStyle, { autocomplete: "(border)-style" }],
   [/^border-([xy])-(.+)$/, handlerBorderStyle],
   [/^border-([rltb])-(.+)$/, handlerBorderStyle],
+  [
+    /^border-([rltb])?-?\[([0-9])\]$/,
```

Comment:
Just curious, why did you choose to limit the range to 9?

Reply:
Hm, good question. It kinda felt natural to me to have this range. Now when I think of it, maybe the arbitrary values should be whatever number? Wdyt?

Reply:
I agree!

Reply:
Allowing now any digit https://github.com/warp-ds/drive/pull/88/commits/3752d88c732ec41d22f22bbe5a23ce8b568e6002